RELEASE-PROCESS 2.md

```diff
@@ -0,0 +1,43 @@
+# Release Process
```

Comment:
@ajanth97 can you remove this document? it appears to be a duplicate of the release process document